Pod/Classes/WavesLayer.swift

```diff
@@ -78,28 +78,35 @@ class WavesLayer: CALayer {
         }
 
         _waveLayers.forEach { layer in
-            let strokeAnimation = CABasicAnimation(keyPath: "strokeEnd")
-            strokeAnimation.duration = 0.3
-            strokeAnimation.fromValue = 1.0
-            strokeAnimation.toValue = 0.0
-            strokeAnimation.isRemovedOnCompletion = false
-            strokeAnimation.fillMode = kCAFillModeForwards
-            layer.add(strokeAnimation, forKey: "strokeEnd")
-
+            
+            animation(for: "strokeEnd", layer: layer)
             if let layer = layer as? RadialGradientLayer {
-                let displayAnimation = CABasicAnimation(keyPath: "opacity")
-                displayAnimation.duration = 0.3
-                displayAnimation.fromValue = 1.0
-                displayAnimation.toValue = 0.0
-                displayAnimation.isRemovedOnCompletion = false
-                displayAnimation.fillMode = kCAFillModeForwards
-                layer.add(displayAnimation, forKey: "opacity")
+
+                animation(for: "opacity", layer: layer)
             }
         }
 
         CATransaction.commit()
     }
 
+    private func animation(for keyPath: String,
```

Comment:
This evokes to me that this method returns `CAAnimation`, I would propose to rename i to `addAnimation`.

Reply:
@mkj-is Done.